SPECS-EXTENDED/frr/frr.spec

```diff
@@ -179,7 +179,7 @@ rm %{buildroot}%{_libdir}/frr/*.so
 rm -r %{buildroot}%{_includedir}/frr/
 
 %pre
-%sysusers_create_compat %{SOURCE2}
+%sysusers_create_package %{name} %{SOURCE2}
```

Comment:
How do we know these two macros do the same thing? I want to make sure we're doing the right thing for this package. Also, since `sysusers_create_compat ` seems to be [a macro still in use by Fedora](https://docs.fedoraproject.org/en-US/packaging-guidelines/UsersAndGroups/), could we instead bring in the package providing it to support the original macro instead? This would also make working with new specs we import from Fedora easier.

Reply:
I don't disagree that we should strive for compatibility, but we'd like to promote this package before the next release and this is the faster option. I'll create an ADO task to track the macro addition.

To answer for Sam: `%sysusers_create_compat` output by definition emulates the behavior of `%sysusers_create_package`, switching out the use of `systemd-sysusers` for `shadow-utils` utilities.

Reply:
Thanks, @oliviacrain! As long as the result is the same, I'm fine with this change.